docs/types/conversion.rst

```diff
@@ -174,15 +174,15 @@ converted to any fixed-size bytes type:
     bytes4 g = 0x0; // fine
 
 String literals and hex string literals can be implicitly converted to fixed-size byte arrays,
-if their number of characters matches the size of the bytes type:
+if their number of characters less or equal to the size of the bytes type:
 
 .. code-block:: solidity
 
     bytes2 a = hex"1234"; // fine
```

Comment:
```suggestion
    bytes2 a = hex"1234"; // not allowed
```

unless I'm missing something?

Reply:
`bytes2` can store 2 bytes. hex"1234" is 2 bytes so this is fine. 
> if their number of characters less or **equal to** the size of the bytes type:

Reply:
Ah, yes, you'd  be right; I was wondering about `bytes2 d = hex"123"; // not allowed`, since 12 bits should fit into 16 bits, however this yields `Error: Expected even number of hex-nibbles.`, which breaks the compilation prior to the implicit conversion step, and thus making the above statements (`if their number of characters is less than or equal to the size of the bytes type:`) true.